module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -473,13 +487,15 @@ public function getSortOptions()
     }
 
     /**
-     * Get an array of sort options for facets.
+     * Get an array of sort options for a facet.
+     *
+     * @param string $facet Facet
      *
      * @return array
      */
-    public function getFacetSortOptions()
+    public function getFacetSortOptions($facet)
```

Comment:
One more question: would it make sense to default `$facet` to `*` for backward compatibility, so it returns default settings if no field is specified?

Reply:
I think that would make this more appropriate for inclusion in release 6.1, vs. waiting until 7.0 for a backward-breaking change.

Reply:
I agree,  this should be backward compatible. Updated the method parameter to have a default value.

Reply:
Thanks!